chainer/sequential.py

```diff
@@ -85,10 +83,12 @@ class Sequential(_link.ChainList):
         as following:
 
         >>> print(model_C)  # doctest: +NORMALIZE_WHITESPACE
-        0       Linear  W(10, 10)       b(10,)
-        1       relu
-        2       Linear  W(10, 10)       b(10,)
-        3       sigmoid
+        Sequential(
+          (0): Linear(in_size=10, out_size=10, nobias=False),
+          (1): <function relu at 0x7f45b1d36510>,
```

Comment:
`doctest: +ELLIPSIS` could be used here to make the test pass.  It seems `NORMALIZE_WHITESPACE` is no longer required.
```suggestion
          (1): <function relu at 0x...>,
```
https://docs.python.org/3/library/doctest.html#doctest.ELLIPSIS